miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/ListUsersControllerST.java

```diff
@@ -0,0 +1,72 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import static org.junit.Assert.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import javax.ws.rs.core.MediaType;
+
+import org.junit.Test;
+
+import com.jayway.jsonpath.JsonPath;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.UserImpl;
+
+public class ListUsersControllerST extends AbstractST {
+
+  private static final String USERS_ENDPOINT = "/admin/users";
+
+  @Test
+  public void testList() throws Exception {
+    String json = testStaticListPage(USERS_ENDPOINT, "data");
+
+    // verify response contains users
+    int userCount = JsonPath.<Integer>read(json, "$.length()");
+    assertTrue("Expected at least one user in response", userCount > 0);
+
+    // spot check first user has required fields
+    assertNotNull(JsonPath.read(json, "$[0].id"));
+    assertNotNull(JsonPath.read(json, "$[0].loginName"));
+    assertNotNull(JsonPath.read(json, "$[0].fullName"));
+    assertNotNull(JsonPath.read(json, "$[0].email"));
+
+    // boolean flags should be present
+    assertNotNull(JsonPath.read(json, "$[0].admin"));
+    assertNotNull(JsonPath.read(json, "$[0].internal"));
+    assertNotNull(JsonPath.read(json, "$[0].active"));
+    assertNotNull(JsonPath.read(json, "$[0].loggedIn"));
```

Comment:
This test is redundant. It's a worse version of the one below because it's only checking for non-null instead of checking the specific values.